paddle/phi/api/yaml/ops.yaml

```diff
@@ -1250,6 +1250,17 @@
   inplace: (x -> out)
   backward : identity_loss_grad
 
+- op : igamma
```

Comment:
According to the [C++ operator development specifications](https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/new_cpp_op_cn.html#qi-kaifasuanzizhuyishixiang), the naming and parameters of operator(C++ API) need to be consistent with Python API, in order to operator can reuse documentation of Python API. When user writes  C++ code directly using operator(C++ API), it will be relatively simple. So the operator needs to be `gammaincc` and `igamma_grad` should be   `gammaincc_grad`

Reply:
done